### DISCUSSION TITLE
There should be a way where we are able to use gqlgen with our choice of router framework and without creating a separate  module.

### DISCUSSION THREAD

**Discussion by piyushgupte** [99designs/gqlgen#1799] [Q&A]
### What happened?
I don't want to create a different module for my graphql service it is part of our multi service architecture so i want to create it within the same module of our project , but when i do "go get github.com/99designs/gqlgen" i get error message due to  conflict in version with the blackfriday package.

go: github.com/russross/blackfriday/v2@v2.0.1 used for two different module paths (github.com/russross/blackfriday/v2 and gopkg.in/russross/blackfriday.v2)

so is there any way we can get rid of this conflicting dependency.

### What did you expect?

There should be a way where we should be able to use graphql gen with our choice of router framework and without creating a separate  module.

### versions
 - `gqlgen version`? => 0.13
 - `go version`?=> 1.14
 - dep or go modules? blackfriday

**Comment by mtibben** [comment]
The module name is `github.com/russross/blackfriday/v2` - see https://github.com/russross/blackfriday/blob/v2/go.mod

Perhaps you can use a [`replace` directive](https://golang.org/ref/mod#go-mod-file-replace)?

**Comment by piyushgupte** [comment]
In my go.mod file i can see replace directive been used , but still i am getting same error on "go get github.com/99designs/gqlgen" am i doing something wrong here.

go.mod====>
replace gopkg.in/russross/blackfriday.v2 => github.com/russross/blackfriday/v2 v2.0.1
